src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1104,6 +1104,9 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         if ($_.Exception -and $_.Exception.WasThrownFromThrowStatement) {
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-macos`
Tests for $ErrorView.ConciseView tests.Activity shows up correctly for scriptblocks
```powershell
Expected like wildcard '*Write-Error: *myError*' to match $null, but it did not match.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/Formatting/ErrorView.Tests.ps1: line 53
53:             $e[2] | Should -BeLike "*Write-Error: *myError*" # wildcard due to VT100
```
Stream writer tests.Redirect Stream Tests.Should write error messages to the error stream
```powershell
Expected ': Testing Error' to be found in file '/private/tmp/98d62fd9-344b-4b00-9837-885d4bb9f4f8/writeoutput.txt', but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1: line 33
33:             $targetfile | Should -FileContentMatch ": Testing Error"
```